README.md

```diff
@@ -93,45 +93,115 @@ Brainflow:
 To use parameters in within VRChat, write the parameter name as a path. For example, to get the left side alpha value, the parameter name would be:

```

Comment:
Change to:

To use these parameters within VRChat, write the parameter name as a path. For example, to get the left side alpha value, the parameter name would be: